docs/site/src/pages/reference/control/sequence-basics.mdx

```diff
@@ -169,4 +172,74 @@ method:
 timed_out = controller.wait_until(lambda sv: sv["pressure_1"] > 25, timeout=10)
 if timed_out:
     print("The condition timed out")
+```
+
+<Divider.Divider direction="x" />
+
+## Blocking While a Condition is True
+
+The `wait_while` method allows you to block the control sequence until a condition is
+no longer true. This method is useful for waiting for a sensor value to reach a certain
+point before continuing. Here's an example of how you can use the `wait_while` method:
+
+```python
+with client.control.acquire(
+    read=["pressure_1", "daq_do_1_state"],
+    write=["daq_do_1_cmd"]
+    write_authorities=[sy.Authority.ABSOLUTE]
+) as controller:
+    # Wait until the pressure sensor value is above 25
+    controller.wait_while(lambda sv: sv["pressure_1"] < 25)
+```
+
+In many ways, you can think of the `wait_while` method as the opposite of the `wait_until`
+method. 
+
+<Divider.Divider direction="x" />
+
+## Asserting a Condition Remains True
+
+The `remains_true_for` method allows you to assert that a condition remains true for a
+certain duration. This method is useful for ensuring that a sensor value remains within
+a certain range for a period of time. Here's an example of how you can use the
+`remains_true_for` method:
```

Comment:
You need to explain what happens if the condition is not true - does the sequence move on? Will the sequence be stuck on this line forever if the `remains_true_for` condition is not met? Be very clear.